lib/components/GalleryCard/SolidBackground.tsx

```diff
@@ -39,26 +40,16 @@ export interface SolidBackgroundProps {
  *
  * @param props Control properties (Defined in `ImageBackgroundProps` interface)
  */
-export const SolidBackground = React.memo((props: SolidBackgroundProps) => {
-    const bgColor = props.backgroundColor ?? defaultColor;
-    const isClass = bgColor ? bgColor.substr(0, 1) !== '#' : false;
-
-    const cls = css(
-        {
-            'background-color': true,
-            'fixed': props.fixed == null || props.fixed,
-            [bgColor]: isClass
-        },
-        props.className
-    );
-
-    const style = isClass 
-        ? { backgroundColor: bgColor }
-        : undefined;
+export const SolidBackground = React.memo(({ fixed, primary, disabled, children, className, attr }: SolidBackgroundProps) => {
+    const cls = css('background-color', { 
+        fixed: fixed == null || fixed, 
```

Comment:
can it be just fixed?

Reply:
what do you mean? This didn't change, to have it not "fixed" you have to explicitly pass "fixed=false" ... if we change this to just be `fixed` you would make it unfixed by default.